.github/workflows/build_hip_basic.yml

```diff
@@ -0,0 +1,53 @@
+name: Build HIP-Basic
+
+on:
+  push:
+    branches: [ develop, release/** ]
+    paths:
+      - 'HIP-Basic/**'
+      - '.github/workflows/**'
+  pull_request:
+    branches: [ develop, release/** ]
+    paths:
+      - 'HIP-Basic/**'
+      - '.github/workflows/**'
+
+env:
+    ROCM_VERSION: 6.2
+    AMDGPU_INSTALLER_VERSION: 6.2.60200-1
+
+jobs:
+    build:
+        name: "Build HIP Examples"
+        runs-on: ubuntu-latest
```

Comment:
Without me looking, I wonder how "ubuntu-latest" interacts with "ubuntu:22.04" a few lines further down. I think this needs a explanation. Perhaps a README file?